src/content/blog/2024_toqito.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Toqito: Quantum Information Science Impact through Open Source"
+author: Vincent Russo and Purva Thakre
+day: 19
+month: 9
+year: 2024
+tags:
+    - python
+    - toqito
+    - microgrant
+    - qis
+---
+
+![](/images/toqito_logo.png)
+
+In early 2020, the humble [initial commit](https://github.com/vprusso/toqito/commit/a2f2449c4b9de81becff41cb25f1bca3fa180e8b) of the toqito project ([repo](https://github.com/vprusso/toqito)|[docs](https://toqito.readthedocs.io/en/latest/index.html)) made its way onto GitHub.
+
+![][/images/toqito_first_commit.png]
+
+toqito was launched as a personal research-oriented software project to help me perform more effective research in quantum information. A  tool like this would have been exceedingly helpful to me as a PhD student as it would have allowed me to rapidly test ideas and accelerate the pace and progress of my research.
```

Comment:
```suggestion
toqito was launched as a personal research-oriented software project to help Vincent perform more effective research in quantum information. A  tool like this would have been exceedingly helpful to Vincent as a PhD student as it would have allowed him to rapidly test ideas and accelerate the pace and progress of his research.
```